dsd/constraints.py

```diff
@@ -68,7 +68,7 @@
 replace_with_close_sequences_key = 'replace_with_close_sequences'
 hamming_probability_key = 'hamming_probability'
 generation_upper_limit_key = 'generation_upper_limit'
```

Comment:
Sorry, for consistency, can we rename `generation_upper_limit` to `num_sequences_to_generate_upper_limit`?

Reply:
Sure!